src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,479 @@
+- **Have a non-trivial number of documents** to manage and fetch (example: a blog with 50+ posts).
+- **Want to enforce frontmatter fields,** and fail if fields are missing (example: every blog post should have a title and description).
+
+## Glossary
+
+We'll be using "schema," "collection," and "entry" throughout this guide. Let's define these terms:
+
+- **Schema:** the shape and type you expect your data to have. In this case, frontmatter data.
+- **Collection:** a set of data that share a common schema. In this case, a set of Markdown and MDX files.
+- **Entry:** A piece of data belonging to a given collection. In this case, a single Markdown or MDX file.
+
+## The content directory
+
+Content Collections introduce a new, reserved directory that Astro will manage for you: `src/content/`. This directory is where all content collections and frontmatter schemas live.
+
+
+## Creating a collection
+
+All entries in `src/content/` **must** be nested in a "collection" directory. This allows you to retrieve a collection of entries based on the directory name, and optionally enforce frontmatter types with a schema. In this way, collections are a bit like database tables or content types in a CMS: they help you group similar kinds of content together.
+
+What this looks like in practice:
+
+```bash "newsletter/" "blog/"
+src/content/
+  # All newsletters have the same frontmatter properties
+  newsletter/
+    week-1.md
+    week-2.md
+    week-3.md
+  # All blog posts have the same frontmatter properties
+  blog/
+    columbia.md
+    enterprise.md
+    endeavour.md
+```
+
+### Organizing with nested directories
+
+Collections are considered **one level deep**, so you cannot nest collections (or collection schemas) within other collections. However, we do allow nested directories to better organize your content. This is vital for certain use cases like internationalization:
+
+```bash
+src/content/
+  docs/
+    # docs schema applies to all nested directories 👇
+    en/
+    es/
+    ...
+```
+
+All nested directories will share the same schema defined for the top-level collection, if any (**docs** in this example).
+
+See [getting from nested directories](#getting-from-nested-directories) to see how folders are treated when retrieving collections.
+
+## Defining a collection schema
+
+Schemas are an optional way to enforce frontmatter types in a collection. To configure schemas, you can create a `src/content/config.{js|mjs|ts}` file. This file should:
+
+1. `export` a `collections` object, with each object key corresponding to the collection's folder name. We will offer a `defineCollection` utility [similar to the `defineConfig` helper](/en/guides/configuring-astro/#the-astro-config-file) (see example below).
+2. Use a [Zod object](https://github.com/colinhacks/zod#objects) to define schema properties. The `z` utility will be built-in and exposed by `astro:content`.
```

Comment:
So hot take: I really liked Chris' edit here 😅 I do prefer how you switched the `collections` export to (2) instead of (1) so I'll use that. I also took his advice to truncate that dashes `-` example into the main example to keep things simple. Let me know if there's further changes you'd like on the latest!